### ISSUE TITLE
Cannot create FileUpload

### ISSUE THREAD

**Issue by fpom** [jupyter-widgets/ipywidgets#2603]
When I create a FileUpload, it doesn't display. I have a bunch of errors in the JS console of the browser. On the attached screenshot, I start with an emptied console, and the restart the kernel & re-run the full notebook. 

I've tried other widgets, they work. 
I've tried with FireFox and got the same errors.

![Screenshot from 2019-11-04 17-07-56](https://user-images.githubusercontent.com/4525552/68136803-28ceea80-ff26-11e9-87a2-c6df0e6a0077.png)

**Comment by michcio1234** [comment]
@fpom did you manage to solve it?

**Comment by fpom** [comment]
Nope, still waiting for some help.

**Comment by michcio1234** [comment]
I tried re-creating my conda virtual env and it didn't help. However, I worked it around by running a notebook server in Docker.

**Comment by fpom** [comment]
That's weird! Probably you don't have the same version (or configuration) of Python inside and outside of Docker.

**Comment by eexwhyzee** [comment]
Also ran into this problem running the notebook locally (macOS), but widget works in docker for me as well (specifically on binderhub/jupyterhub deployments on k8s)

**Comment by martinRenou** [comment]
The `FileUpload` widget is quite new, and I suspect you are in an environment where you updated ipywidgets at some point. I would suggest recreating a new environment and make a clean installation of ipywidgets.

Also, you can try restarting the server/refreshing the page after updating ipywidgets.

**Comment by michcio1234** [comment]
@martinRenou not really, as I wrote a few comments above, I tried creating a new conda virtual env with everything installed from scratch and it didn't help.

**Comment by martinRenou** [comment]
Could it happen that you have ipywidgets or Jupyter installed in the base conda environment? Which would conflict with your current environment.

**Comment by martinRenou** [comment]
Also, if it's JupyterLab, do not forget to re-install the extension with:
```
jupyter labextension install @jupyter-widgets/jupyterlab-manager
```
